### PR TITLE
Replacing empty tables with empty state message

### LINKED ISSUE #109: See whether "No <whatever> to show" messages look better outside of the tables
Original issue https://github.com/openshift/origin/issues/5412

Per @spadgett comment, there is an empty state for tables in patternfly:
https://www.patternfly.org/patterns/table-view/

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes #109
Replaces #497

HACK DAY!

Before:
127 0 0 1-8443-console-project-my-project-dc-browse-deployments
127 0 0 1-8443-console-project-my-project-dc-browse-deployments 1

After:
localhost-9000-dev-console-project-my-project-empty-quota
localhost-9000-dev-console-project-my-project-delete-browse-config-maps-labelfilter 7b 22matchexpressions 22- 5b 7b 22key 22- 22foo 22 22operator 22- 22exists 22 22values 22- 5b 5d 7d 5d 7d

### CODE REVIEW

File: app/views/browse/config-maps.html [hunk 1]
```diff
@@ -26,38 +26,46 @@
...
-                    <span am-time-ago="configMap.metadata.creationTimestamp"></span>
-                  </td>
-                  <td data-title="Labels">
-                    <em ng-if="(configMap.metadata.labels | hashSize) === 0">none</em>
-                    <labels labels="configMap.metadata.labels" clickable="true" kind="Config Map" project-name="{{configMap.metadata.namespace}}" limit="3" filter-current-page="true"></labels>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+          <div ng-if="(configMaps | hashSize) == 0">
```
Comment (dtaylor113) [inline app/views/browse/config-maps.html]: Hi, would be great to use the [pfEmptyState](https://www.patternfly.org/angular-patternfly/#/api/patternfly.views.component:pfEmptyState) component.

**Comment by spadgett** [comment]
@rhamilto I don't see the `$scope.filterWithZeroResults` value being reset to `false` in most components. Is that a problem?

**Comment by rhamilto** [comment]
> @rhamilto I don't see the $scope.filterWithZeroResults value being reset to false in most components. Is that a problem?

It did not appear to be in my testing.  Is there a better way?

File: app/views/browse/config-maps.html (hunk 1 above)
Comment (rhamilto) [reply app/views/browse/config-maps.html]: @dtaylor113, I opted to use the pre-existing empty-state styles that pre-date pfEmptyState that were already in use in the console for the sake of consistency.

**Comment by benjaminapetersen** [comment]
Nice, def visually is an improvement.

**Comment by rhamilto** [comment]
Revised to replace filter alert with empty state message containing "Clear Filter" link and addressing comment from @spadgett.  PTAnotherL.

**Comment by rhamilto** [comment]
Filter button is labeled "Clear filters".  Empty state link is labeled "Clear Filter" as that's what we're using on the overview.  These should be consistent?

**Pull request merged**